src/components/text/TextBit.jsx

```diff
@@ -44,7 +45,7 @@ const TextBit = ({
   ...props}: TextBitPropsType) => {
   const Type = type;
   const textClass = classNames('sg-text-bit', {
-    [`sg-text-bit--${size || ''}`]: size,
```

Comment:
What if size is undefined? Shouldn't it be `size && size !== TEXT_BIT_SIZE.NORMA`